engine/app/components/citizens_advice_components/breadcrumbs.html.haml

```diff
@@ -1,10 +1,11 @@
-%nav{ class: "cads-breadcrumbs cads-breadcrumbs--#{type.to_s.dasherize}", "aria-label": "breadcrumbs" }
-  %ul.cads-breadcrumbs__list
-    - links.each_with_index do |link, index|
-      %li.cads-breadcrumbs__crumb
-        - if index == links.size - 1
-          %span.cads-breadcrumb{ aria: { current: "location" } }
-            = link[:title]
-        - else
-          %a.cads-breadcrumb{ href: link[:url] }
-            = link[:title]
+- haml_tag_if @full_width, ".cads-breadcrumbs-wrapper" do
```

Comment:
By convention we might want to define a `full_width?` method on the component rather than using `@full_width`.